cds-snc/cppd-infrastructure
# Main services

# Description

When performing a `terraform apply`, the connection strings to redis and postgres will get added to the key vault, making it easier to tear down and stand up the infrastructure.

# Test Instructions

1. Perform a `terraform apply`
2. Once complete, check the key vault for updated values for redis and postgres
